codalab/bundles/run_bundle.py

```diff
@@ -36,6 +36,11 @@ class RunBundle(DerivedBundle):
     METADATA_SPECS.append(MetadataSpec('request_priority', int, 'Job priority (higher is more important). Negative '
                                                                 'priority bundles are queued behind bundles with no specified priority.', default=None))
     METADATA_SPECS.append(MetadataSpec('request_network', bool, 'Whether to allow network access.', default=False))
+    METADATA_SPECS.append(
+        MetadataSpec('cpu_usage', float, 'Portion of CPU used by this container (e.g., 0.24)', default=0.0, generated=True))
+    METADATA_SPECS.append(
+        MetadataSpec('memory_limit', int, 'Limit of Memory available to this container (e.g., 2085326848)', default=0, generated=True))
```

Comment:
I'm a bit confused by what memory_limit actually means, according to this description. Can you look into this and update the description to be clearer about what is actually means?

Reply:
sgtm